File: Python/URI_1012.py
pi = 3.14159

lista = input().split(' ')
A = float(lista[0])
B = float(lista[1])
C = float(lista[2])

TRIANGULO = (A * C) / 2.0
CIRCULO = pi * C**2
TRAPEZIO = ((A + B) * C) / 2
QUADRADO = B * B
RETANGULO = A * B

print("TRIANGULO: %0.3F" % TRIANGULO)
print("CIRCULO: %0.3F" % CIRCULO)
print("TRAPEZIO: %0.3F" % TRAPEZIO)
print("QUADRADO: %0.3F" % QUADRADO)
print("RETANGULO: %0.3F" % RETANGULO)

File: C++/URI_1012.cpp
#include <iostream>
#include <iomanip>
 
using namespace std;
 
int main() {
 
    float A, B, C;
    cin >> A >> B >> C;
    
    cout << fixed << setprecision(3);
    cout << "TRIANGULO: " << (A*C)/2 << endl;
    cout << "CIRCULO: " << C*C*3.14159 << endl;
    cout << "TRAPEZIO: " << ((A+B)*C)/2 << endl;
    cout << "QUADRADO: " << B*B << endl;
    cout << "RETANGULO: " << A*B << endl;
 
    return 0;
}